app/assets/stylesheets/style.css

```diff
@@ -528,6 +540,12 @@ a.btn-outline-secondary .fa {
   color: #6c757d;
 }
 
+.container{
```

Comment:
Hmm, I'm a little worried about modifying default Bootstrap styles here, and about modifying the HTML and body styles... Is there a more minimal way to do this, or is this the only way? 

Would it work to just absolute position the footer? And, I guess, add corresponding amount of bottom-padding to the body so it doesn't overlap anything?

Thanks for working through this with us, just trying to be sure it works on a variety of screen sizes and is maintainable and Bootstrap compatible! Thanks a lot!